charts/crowdsec/values.yaml

```diff
@@ -56,6 +56,11 @@ lapi:
       cpu: 150m
       memory: 100Mi
 
+  metrics:
+    enabled: false
+    serviceMonitor: 
```

Comment:
Maybe just add few word that explain what needed to enable this feature and add link to help on how to enable serviceMonitor.

Reply:
I added the official documentation and also another link which helped me more than the official documentation.